utils.go

```diff
@@ -26,14 +26,37 @@ func countLines(file_name string, ext string) (int32, int32, int32, int32) {
 	var code int32 = 0
 	var gap int32 = 0
 	var comments int32 = 0
+	var inside_multi_line_comment bool =false
+	var multi_comment_str_open string=""
+	var multi_comment_str_close string=""
 	comment_str, exists := comment_map[ext]
+	multi_comment_str_pair,multi_exists :=multi_comment_map[ext]  
+	//multi_comment_str_pair will have the opening and closing symbols ' : ' separated
+	if multi_exists{
+		multi_comment_str_open=strings.Split(multi_comment_str_pair,":")[0]
+		multi_comment_str_close=strings.Split(multi_comment_str_pair,":")[1]
+	}
 	for {
 		content, _, err := reader.ReadLine()
 		content_str := string(content[:])
 		if err != nil {
 			break
 		}
-		if content_str == "" {
+		//Checks if [Opening symbol] is present at staring of the line
+		if multi_exists && strings.HasPrefix(strings.TrimSpace(content_str),multi_comment_str_open){
+			inside_multi_line_commentt=true
```

Comment:
"inside_multi_line_commentt" variable is not defined. Please check @yogesh19k. There is a typo...